lib/phoenix_component.ex

```diff
@@ -2710,6 +2710,14 @@ defmodule Phoenix.Component do
     """
   )
 
+  attr.(:skip_persistent_id, :boolean,
```

Comment:
What about `persistent_ids: boolean()` and the default is true? Either way is fine, your choice.

Reply:
Gah, ignore me, we already have `skip_hidden`, so we should be consistent. Ship it!!!